subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/UiTestCheck.kt

```diff
@@ -15,6 +15,11 @@ open class UiTestCheck(context: String) : SuppressibleBuildStep(context),
 
     var configurations = mutableListOf<String>()
 
+    /**
+     * Whether skip results of tests marked with @Flaky or not
+     */
+    var suppressFlaky: Boolean = false
```

Comment:
FYI: could be an interface like `impactAnalysisAware`

Reply:
Done. Create an interface `FlakyAwareBuildStep`.